### ISSUE TITLE
[Scraper] scraping with adultdvdempire: actors do not work with latest git build

### ISSUE THREAD

**Issue by waywit** [Komet/MediaElch#1011]
**Describe the bug**
After scraping movies with adultdvdempire scraper no actors nor the pictures are scraped. This time I have also run the tests, and the scraper tests say:
```
../test/scrapers/testAdultDvdEmpire.cpp:61: FAILED:
  REQUIRE( actors.size() > 15 )
with expansion:
  0 > 15
```
Perhaps something has changed on the site? But for me it looks like it may match to the regex:

```html
...
title="Magic Mike XXXL"><span class="overlay overlay-preview"><i class="fa fa-search-plus"></i></span></a></div></div><div class="m-b-1"><strong>Starring:</strong>&nbsp;

<a href="/652646/adriana-chechik-pornstars.html"
	style="display:inline-block;"
	Category="Item Page" Label="Performer"><div class="hover-popover-container"><u>Adriana Chechik</u><div class="hover-popover hover-popover-right"
					style=""><div class="hover-popover-detail" style="padding:0;"><img src="https://imgs1cdn.adultempire.com/actors/652646h.jpg"
							alt="Adriana Chechik Bodyshot" 
							title="Adriana Chechik" 
							class="img-responsive" /></div></div></div></a>, 

<a href="/294032/india-summer-pornstars.html"
	style="display:inline-block;"
	Category="Item Page" Label="Performer"><div class="hover-popover-container"><u>India Summer</u><div class="hover-popover hover-popover-right"
					style=""><div class="hover-popover-detail" style="padding:0;"><img src="https://imgs1cdn.adultempire.com/actors/294032h.jpg"
							alt="India Summer Bodyshot" 
							title="India Summer" 
							class="img-responsive" /></div></div></div></a>, 
...
```

**To Reproduce**
1. Use MediaElch 6.7.dev to scrape "Magic Mike XXXL" with adultdvdempire scraper
2. Check if the actors are loaded in UI and thumbs were downloaded (if activated)

or

Run the scraper tests.

**Expected behavior**
All actors were scraped correctly and thumbs were downloaded to ".actors" directory.

or

Scraper tests rn without an error

**Screenshots**
No.

**MediaElch Version:**
 - [X] 2.6.7-dev (compiled from source)
 - [ ] 2.6.6 stable

**Operating System:**
 - [ ] Windows
 - [ ] macOS
 - [X] Linux (distribution Ubuntu: 20.09)

**Additional context**
Here you can find the full log file from scraper tests:
[MediaElch_scraper_test.log](https://github.com/Komet/MediaElch/files/5189308/MediaElch_scraper_test.log)

**Comment by bugwelle** [comment]
Thanks! I will look into it. I haven't run the adult scraper tests in a while as I'm mostly working on my company's laptop.

**Comment by bugwelle** [comment]
Will be fixed by #1012 

The reason it broke: New `<div>` elements were added. Also, if we use a User-Agent that contains "MediaElch", we get different HTML without links to images. We now use a Firefox header.
